api.swagger.yaml

```diff
@@ -5471,6 +5471,25 @@ paths:
           description: |
             The notification does not exist.
       description: Mark a notification as read.
+  '/notifications/clear_all':
+    parameters:
+      - description: Marks all notifications for the mobile app as read. If true, this will mark as read notifications with entity type user, and entity type node and bundle type post only.
+        in: query
+        name: app_notifications_only
+        schema:
+          type: boolean
+          default: false
+    post:
+      tags:
+        - notifications
+      responses:
+        '204':
+          description: The user marked the notifications as read.
+        '403':
+          description: User is not authorized to clear all notifications.
+        '404':
+          description: Could not find user to corresponding uuid.
```

Comment:
This doesn't take a UUID parameter so does it really make sense to return a 404 here? If somehow the _current_ user doesn't exist, wouldn't that just be a 403?